front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/BannerImageFields/SelectPreviewDevice.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Would it make sense to move it to `app/components/admin` as we use it in different parts of the project?

Reply:
Yes, certainly